helm/superset/templates/init-job.yaml

```diff
@@ -83,6 +83,32 @@ spec:
         command: {{  tpl (toJson .Values.init.command) . }}
         resources:
           {{- toYaml .Values.init.resources | nindent 10 }}
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if or .Values.affinity .Values.init.affinity }}
+      affinity:
+        {{- with .Values.affinity }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with .Values.init.affinity }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      {{- end }}
+      {{- if or .Values.topologySpreadConstraints .Values.init.topologySpreadConstraints }}
+      topologySpreadConstraints:
+        {{- with .Values.topologySpreadConstraints }}
```

Comment:
Do we have default values for these in `values.yaml`? (`topologySpreadConstraints` and `init.topologySpreadConstraints`)

Reply:
The default values for `topologySpreadConstraints` and `init.topologySpreadConstraints` is `[]`.